### ISSUE TITLE
Cards not very readable with NC 14 black theme

### ISSUE THREAD

**Issue by HorlogeSkynet** [nextcloud/deck#645]
Hey over here !

**Describe the bug**
Simple feedback of a short usage of this wonderful app, once the Nextcloud 14 black theme has been enabled : Cards are not really readable...

**To Reproduce**
1. Run the Deck application
2. Choose a board
2. See the unreadable aspect of cards

**Expected behavior**
A cards design compatible with the black theme.

**Screenshots**
![image](https://user-images.githubusercontent.com/5331869/46854534-6df71a80-ce01-11e8-8339-12568322acf9.png)

**Client details:**
 - OS: Debian 10 (_Buster_) 
 - Browser: Firefox
 - Version: v62.0.3
 - Desktop environment: Cinnamon (with a dark theme too)
 - Device: Laptop

<details>
<summary>Logs</summary>

#### Browser log
```
Content Security Policy: The page’s settings blocked the loading of a resource at self (“script-src”). Source: (()=>{
		let ODP=(t,p,o)=>{
			try
			{
.... deck:1
```
</details>

Thanks bye :wave:

**Comment by juliushaertl** [comment]
Thanks for reporting. I have already started fixing this in #643

**Comment by HorlogeSkynet** [comment]
Wow such a fast feedback, thanks to you and sorry then (your PR does not contain "black" nor "theme") :joy:

++ :bowing_man:

**Comment by juliushaertl** [comment]
@HorlogeSkynet Let's keep it open, so we can properly keep track of it :wink:

**Comment by muppeth** [comment]
Although merged and fixed, it looks like (at least since deck 1.0 and nextcloud 18.04) the cards are yet again indistinguishable from the background. Also the line seperating the columns has gnone missing presumable using the same color as backgorund. 
![Screenshot from 2020-05-09 09-47-13](https://user-images.githubusercontent.com/8480830/81467630-45502600-91da-11ea-80ad-1354a5d47c3e.png)
Additionally, in the list of project, IMO the line seperating projects seems to be to thick.
![Screenshot from 2020-05-09 09-46-17](https://user-images.githubusercontent.com/8480830/81467631-46815300-91da-11ea-8771-0fe87b9a4247.png)
I think it has to do with the fact, dark mode's global background used to be a bit brighter.
I don't know if that's stepping in with shoes in your house, but perhaps I could play around with css and submit some ideas if you don't mind.

**Comment by juliushaertl** [comment]
@muppeth Suggestions how to increase the visibility of the cards are very welcome. Feel free to open a new issue to discuss this in detail or just a pull request with some css changes :wink:
